docs/conformance/README.md

```diff
@@ -0,0 +1,192 @@
+
+  management_cidrs = [
+    "0.0.0.0/0",       # Instances can be SSH-ed into from anywhere on the internet.
+  ]
+
+  # This is different for each project on Packet and depends on the packet facility/region. Check yours from the `IPs & Networks` tab from your Packet.net account. If an IP block is not allocated yet, try provisioning an instance from the console in that region. Packet will allocate a public IP CIDR.
+  node_private_cidr = "..."
+}
+
+module "worker" {
+  source = "git::https://github.com/kinvolk/lokomotive-kubernetes//packet/flatcar-linux/kubernetes/workers?ref=master"
+
+  providers = {
+    local    = "local.default"
+    template = "template.default"
+    tls      = "tls.default"
+    packet   = "packet.default"
+  }
+
+  ssh_keys = [
+    "...",
+  ]
+
+  # Packet
+  cluster_name = "conformance-cluster"
+  project_id   = "..."
+  facility     = "ams1"
+
+  pool_name    = "worker"
+
+  count = 2
+  type  = "t1.small.x86"
+
+  kubeconfig = "${module.controller.kubeconfig}"
+
+  labels = "node.conformance.io/role=backend,node-role.kubernetes.io/backend="
+}
+```
+
+Once the cluster is created, configure your kubectl and make sure you can connect to it with kubectl:
+
+```
+$ export KUBECONFIG="$PWD/assets/conformance-cluster/auth/kubeconfig"
+$ kubectl get nodes
+NAME                                  STATUS   ROLES               AGE   VERSION
+conformance-cluster-controller-0      Ready    controller,master   71s   v1.15.3
+conformance-cluster-worker-worker-0   Ready    backend,node        71s   v1.15.3
+conformance-cluster-worker-worker-1   Ready    backend,node        71s   v1.15.3
+```
+
+Download latest binary release of sonobuoy:
+
+```
+wget https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.16.2/sonobuoy_0.16.2_linux_amd64.tar.gz
+wget https://github.com/vmware-tanzu/sonobuoy/releases/download/v0.16.2/sonobuoy_0.16.2_checksums.txt
+sha256sum --ignore-missing -c sonobuoy_0.16.2_checksums.txt
+tar xvf sonobuoy_0.16.2_linux_amd64.tar.gz
+```
+
+Run sonobuoy in conformance mode:
+
+```
+./sonobuoy run --mode=certified-conformance
+```
+
+Wait for sonobuoy to finish (might take a long time). You can check its status by running:
+
+```
+./sonobuoy status
+```
+
+Once the run shows as `completed`, copy the output to a local directory:
+
+```
+outfile=$(./sonobuoy retrieve)
+```
+
+Now you can extract the results snapshot:
+
+```
+mkdir ./results; tar xzf $outfile -C ./results
+```
+
+Finally, clean up the Kubernetes objects created by Sonobuoy:
+
+```
+./sonobuoy delete
+```
+
+## Make a PR to get into k8s-conformance
+
+Check the tests pass by going to the end of the file `results/plugins/e2e/results/global/e2e.log` and, if so, create a PR with the results as explained in the [documentation](https://github.com/cncf/k8s-conformance/blob/master/instructions.md#uploading).
+
+For `PRODUCT.yaml`, you can use the following values:
+
+```
+vendor: Kinvolk GmbH
+name: Lokomotive Kubernetes
+version: $LOKOMOTIVE_VERSION
```

Comment:
What should version be? I see you used `master` in the tf files, shouldn't we at least use a commit hash?

Reply:
Yes. I think having `master` in documentation is fine since having an arbitrary commit hash seems weird. What I thought about was clarifying that I'm using master in this document but for submissions we should use the corresponding version. Would that be ok?

Reply:
Oh, yes, makes sense to me :)

Reply:
Updated.